.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.209.6/containers/docker-existing-dockerfile
+{
+	"name": "VS Code extension of Poetry Docker dev image",
+
+	// Base on Dockerfile one level up:
+	"context": "..",
+	"dockerFile": "../Dockerfile",
+
+	// Mount to WORKDIR as specified in Dockerfile:
+	"workspaceMount": "source=${localWorkspaceFolder},target=/home/dev/src,type=bind",
+	"workspaceFolder": "/home/dev/src",
+
+	// Python IDE settings:
+	"settings": {
+		"python.pythonPath": "/home/dev/poetry-virtualenvs/modelon-impact-client-1JqLc1Zx-py3.7/bin/python",
```

Comment:
Yes, Poetry: "...code deduces the location of the environment by using the project name from pyproject.toml and adding the hash of the parent directory of pyproject.toml". So it will be the same as we don't change the WORKDIR.

Reply:
Ah that sounds safe! 👍